internal/gitlab/gitlab.go

```diff
@@ -251,31 +251,26 @@ func ApplyGitlabTerraform(dryRun bool, directory string) {
 		}
 		//* AWS_SDK_LOAD_CONFIG=1
 		//* https://registry.terraform.io/providers/hashicorp/aws/2.34.0/docs#shared-credentials-file
-		os.Setenv("AWS_SDK_LOAD_CONFIG", "1")
-		os.Setenv("AWS_PROFILE", config.AwsProfile)
+		envs := map[string]string{}
```

Comment:
we could have less variables using `config.EnvVariable`, and AFAIK `viper` is global, we can just call `viper.GetString("value")`

Reply:
I am not aware of a better way to address it, I used the same standard of tools that create HTTP requests with headers.

A map is provided and the framework translates into headers or vars or post body, and so on.